rm-datasource/src/main/java/io/seata/rm/datasource/sql/struct/cache/MysqlTableMetaCache.java

```diff
@@ -65,6 +70,35 @@ public static TableMetaCache getInstance() {
         return tableMetaCache;
     }
 
+    @Override
+    protected String getCacheKey(DataSourceProxy dataSourceProxy, String tableName) {
+        StringBuilder cacheKey = new StringBuilder(dataSourceProxy.getResourceId());
```

Comment:
@l81893521 Does tableName need to remove the database name here?

Reply:
Thanks. Fixed.